aten/src/ATen/native/Resize.cpp

```diff
@@ -16,8 +16,14 @@ Tensor& resize_cpu_(Tensor& self, IntArrayRef size) {
   return self;
 }
 
-Tensor& resize_as_cpu_(Tensor& self, const Tensor& the_template) {
-  Tensor& result = resize_cpu_(self, the_template.sizes());
+// I am a naughty man
```

Comment:
Please clarify? It's just that we shouldn't be resizing sparse tensors, right?

Reply:
Oh, it's because I stuck the forward declaration here, instead of in a header

Reply:
It's soooo naughty that ROCm hates it. I guess I'll need to find another way to do this.